doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -79,6 +79,7 @@ Bluetooth Mesh
   * The Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD` to no longer depend on the Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA`.
   * The Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_CLI` to no longer enable the Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD` by default.
     The Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD` can still be manually enabled.
+  * The JSON file added to ``dfu_application.zip`` when metadata is automatically generated by enabling the Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD` to now contain a field for the ``core_type`` used when encoding the metadata.
```

Comment:
Oki, a suggestion :)
```suggestion
  * The JSON file, added to :file:`dfu_application.zip` during the automatic DFU metadata generation, to now contain a field for the ``core_type`` used when encoding the metadata.
    Automatic metadata generation is enabled by enabling the Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD`. 
```

Reply:
The second sentence is not really needed I think, since what that sentence describes has not really changed? Especially since basically the same thing (The Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD` can still be manually enabled) is written in the bullet immediately above. It would become very repetetive to mention this every time we talk about automatic metadata generation (which was a new feature in the previous release, so it's not like people haven't heard of it at all by the time they read this changelog)

Reply:
Can I keep just the first half of your suggestion?